### ISSUE TITLE
Question -ModuleNotFoundError: No module named 'pygimli'

### ISSUE THREAD

**Issue by AbuMoaz** [gimli-org/gimli#160]
Hello, everyone,
I installed (pygimli-1.0.9-cp36-cp36m-win_amd64.whl) successfully by Anaconda3. But, when I wanted running an example by Pycharm got this error (ModuleNotFoundError: No module named 'pygimli').
Please, can anyone help me? Thank you in advance!

### My environment
**Windows**
**Python version3.6**
**pyGIMLi 1.0.9**
**Anaconda3**

**Comment by AbuMoaz** [comment]
@Coastal0 thank you for your reply and help!
I got this the error when I tried to follow the link instruction. Maybe I did something wrong.

C:\Users\Mosaad\AppData\Local\conda\conda\envs\mo1\python.exe C:/Users/Mosaad/AppData/Local/conda/conda/envs/mo1/Lib/site-packages/plot_mod-dc-2d.py
ERROR: cannot import the library '_pygimli_'.
No module named 'pygimli.core._pygimli_'
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\pygimli-1.0.9-cp36-cp36m-win_amd64.whl\pygimli\core\__init__.py", line 22, in <module>
    from ._pygimli_ import *
ModuleNotFoundError: No module named 'pygimli.core._pygimli_'
Traceback (most recent call last):
  File "C:/Users/Mosaad/AppData/Local/conda/conda/envs/mo1/Lib/site-packages/plot_mod-dc-2d.py", line 34, in <module>
    import pygimli as pg
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "C:\ProgramData\Anaconda3\pygimli-1.0.9-cp36-cp36m-win_amd64.whl\pygimli\__init__.py", line 52, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "C:\ProgramData\Anaconda3\pygimli-1.0.9-cp36-cp36m-win_amd64.whl\pygimli\core\__init__.py", line 32, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "C:\ProgramData\Anaconda3\pygimli-1.0.9-cp36-cp36m-win_amd64.whl\pygimli\_logger.py", line 91, in <module>
  File "C:\ProgramData\Anaconda3\pygimli-1.0.9-cp36-cp36m-win_amd64.whl\pygimli\_logger.py", line 76, in setDebug
AttributeError: 'NoneType' object has no attribute 'setDebug'

**Comment by carsten-forty2** [comment]
why do you use: `C:\Users\Mosaad\AppData\Local\conda\conda\envs\mo1\python.exe`
instead of the `python.exe` shipped with anaconda.

I'm not familiar with pycharm. Does the 'import pygimli' work within the spyder ide?

**Comment by halbmy** [comment]
I installed pyCharm (a 30-day test version of the software that looks good for 200$) and checked it out. By default, it uses a new conda/virtualenv/pipenv environment for each project, which is actually a good idea. But projects can also be configured to work with the base environment, e.g. Anaconda3/python.exe.

In any case one cannot import simple modules like numpy as there are dll files required in the path Anaconda3/Library/bin that are not found if they are not included in the path. This also holds if you call Spyder.bat/exe directly, but NOT if you call Spyder from the Windows start menu entry. At least not with the two (quite recent) Anaconda distributions I have tested. Can you try that?

At any rate, it is NOT a pyGIMLi problem but one with your Python distribution, which is why I close the issue. You will find issues about that in the internet.

**Comment by halbmy** [comment]
Dear Abu,
sorry for being a bit rude without any reason (your issue was about something completely different and I mixed things up). I had just installed pyCharm and got the same problem that other people got with the latest Anaconda distribution installers (i.e. that setting the path to Anaconda is not enough anymore), thus being frustrated about things we cannot change but that confuse users like you. 
Your feedback is highly welcome.

**Comment by AbuMoaz** [comment]
@halbmy @florian-wagner 
Now, It works seamlessly after use the updated example with the spyder. I hope to support me with the source of the updated examples or tutorials that related to IP borehole and DC resistivity. Thank you so much for you and all the respected team of the PyGimli code.

![captureupex](https://user-images.githubusercontent.com/40517803/52125994-0e02d900-2669-11e9-9f3a-ea166538e3b2.PNG)

**Comment by florian-wagner** [comment]
@AbuMoaz Glad to hear that it works with Spyder. If you install Gmsh (www.gmsh.info), the example should produce a figure, too. 

And we will soon provide ERT / IP imaging examples on the website. For now you could install the pybert wheel as well: https://gitlab.com/resistivity-net/bert and have a look at the simple 2D ERT modeling and inversion example: https://gitlab.com/resistivity-net/bert/blob/master/examples/pybert/2d-python-simple/simple.py

**Comment by AbuMoaz** [comment]
@florian-wagner 
Ok, thank you for your help!

**Comment by sanaamouzahir** [comment]
> @halbmy @florian-wagner Now, It works seamlessly after use the updated example with the spyder. I hope to support me with the source of the updated examples or tutorials that related to IP borehole and DC resistivity. Thank you so much for you and all the respected team of the PyGimli code.
> 
> ![captureupex](https://user-images.githubusercontent.com/40517803/52125994-0e02d900-2669-11e9-9f3a-ea166538e3b2.PNG)

Please could you summarize the steps you have taken ? I am trying to do the same as you but I still get the 'no module named pygimli' error.
